sci-libs/cantera/cantera-2.4.0-r3.ebuild

```diff
@@ -32,7 +32,7 @@ RDEPEND="
 			dev-python/numpy[${PYTHON_MULTI_USEDEP}]
 		')
 	)
-	<sci-libs/sundials-5.1.0:0=
+	<sci-libs/sundials-5.2.0:0=
```

Comment:
This needs to be revbumped, but you can just commit to stable. Or in this case, use `git mv`.

Reply:
"git mv" is made from -r3 to -r4 with saving stabilization.